### ISSUE TITLE
converting shared notes to inline notes

### ISSUE THREAD

**Issue by HonkXL** [fisharebest/webtrees#1163]
webtrees becomes very slow if a tree is imported with a lot of shared notes. It would be a great feature to have an option in the batch-update to convert the shared notes to inline notes.

I think this should only run for notes that are linked to one INDI or one family. Then it would be great to have a mass-deletion of shared notes that are linked to 0 objects.

See dicussion here:

https://www.webtrees.net/index.php/en/forum/help-for-ver-1-7-8/31726-shared-notes-are-very-slow

**Comment by hartenthaler** [comment]
Does this issue still make sense (when looking in the direction of GEDCOM 7)? Maybe the other way around makes more sense now: using a data fix to convert inline notes to shared notes.

**Comment by miqrogroove** [comment]
I don't think this is a good solution for performance issues.  It would be better to resolve the performance issues, and then if you still need a tool to create inline notes then that would be something to consider.  So, do we have an open issue for List Performance??

I have about 1,800 shared notes now and it does take several seconds to display the first 10 notes.

Would it be possible to disable the privacy checks for the List, and then simply throw an error if someone clicks a note that they're not allowed to view?
